index.js

```diff
@@ -163,4 +178,30 @@ PagedContacts.authorized = PagedContactsModule.authorized;
 if (Platform.OS === 'ios') {
   PagedContacts.notDetermined = PagedContactsModule.notDetermined;
   PagedContacts.restricted = PagedContactsModule.restricted;
-}
\ No newline at end of file
+}
+
+/**
+ * The `PagedContactsImage` class is a simple wrapper around the `Image` component
+ * using `PagedContacts.setImageViewWithHandle`
+ * function to update the view with the contact picture on the native side.
+ * 
+ * @prop {Number} contactId Contact identifier
+ * @prop {String} imageType Image type - either `image` or `thumbnailImage`
+ * 
+ * @memberOf PagedContacts
+ */
+class PagedContactsImage extends Component {
```

Comment:
I don't think Clubs upload the images for now. If they will, I think we can do something similar with uploading (it can be more generic and elegant, but for now we were looking for a quick fix).

By the way, thumbnails are huge in iOS actually, so we do need to resize them, but it's not related to this change.